src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StorageDaoImplTest {
```

Comment:
Seems u need to test also age = 18 according to common mistake. 18 is edge parameter.

Reply:
done